Project.toml

```diff
@@ -3,11 +3,25 @@ uuid = "0004c1f4-53c5-4d43-a221-a1dac6cf6b74"
 authors = ["Jose Storopoli <thestoropoli@gmail.com> and contributors"]
 version = "0.1.0"
 
+[deps]
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+StatsModels = "3eaba693-59b7-5ba5-a881-562e759f1c8d"
+Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
+Turing = "fce5fe82-541a-59a6-adf8-730c64b5f9a0"
+
 [compat]
+Reexport = "1.2"
+StatsModels = "0.6"
+Tables = "1.6"
+Turing = "0.19"
 julia = "1.6"
 
 [extras]
+CategoricalArrays = "324d7699-5711-5eae-9e2f-1d82baa6b597"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
-test = ["Test"]
```

Comment:
Don't use this old way of adding test extras. Add a separate project to `test/`. See for example https://github.com/JuliaBooks/Books.jl/tree/main/test or most projects inside TuringLang

A separate Project.toml is nicer for two reasons:

1. You can do `pkg> activate test` to get all test deps
2. you can set CompatHelper to check the test deps (not super important, but can be nice)

Reply:
Thanks!